backend/alembic/env.py

```
@@ -2,8 +2,7 @@
 import sys
 from logging.config import fileConfig
 
-from sqlalchemy import engine_from_config
-from sqlalchemy import pool
+from sqlalchemy import engine_from_config, pool
```

Flake8 (7.2.0)

[error] 'sqlalchemy.engine_from_config' imported but unused

(F401)